allensdk/api/queries/brain_observatory_api.py

```diff
@@ -427,6 +430,41 @@ def save_ophys_experiment_event_data(self, ophys_experiment_id, file_name):
 
         self.retrieve_file_over_http(self.api_url + file_url, file_name)
 
+    @staticmethod
+    def save_ophys_experiment_eye_tracking_data(
+            ophys_experiment_id,
+            cloud_cache: S3CloudCache
+    ) -> Path:
+        """
+        Downloads eye tracking data for `ophys_experiment_id` using
+            `S3CloudCache`
+        S3 used instead of warehouse due to no experience on current team
```

Comment:
You can also say that we don't expect any future datasets to use the warehouse.

Reply:
I reworded it.